### PR TITLE
Dashboard nav skeleton

### PR DESCRIPTION
**Pull request by pmernyei**
Basic navigation for dashboard with placeholder components declared for 3 components. I'm pretty sure the way I coded the navigation is a mess (storing the selected component as a string and casing a huge ternary expression on it), I'm not really comfortable with JSX syntax, feel free to either a) suggest how to do properly, or b) just merge quickly so we can throw stuff in?
image

### CODE REVIEW

File: Dashboard/src/exampleData.js
Comment (gezacsenger) [inline Dashboard/src/exampleData.js:9]: /me likes the name.

**Review by CsengerG — APPROVED** [review]
I don't see anything wrong with this. I don't think it's worth our time to refactor or change code at this early stage as it will change anyway. So I'll approve to unblock you (+ other people).

For future reference: I don't know what the units for width/heights are in React (I see that somewhere you used pixels, somewhere it's vh, etc.) but somewhere it's unitless. AFAIK people like to use consistent measures, so we should decide early what we want (I think that would help keeping this whole thing more manageable, but maybe React handles everything well, I don't know... this might be bollocks).

**Comment by pmernyei** [comment]
> For future reference: I don't know what the units for width/heights are in React (I see that somewhere you used pixels, somewhere it's vh, etc.) but somewhere it's unitless. AFAIK people like to use consistent measures, so we should decide early what we want (I think that would help keeping this whole thing more manageable, but maybe React handles everything well, I don't know... this might be bollocks).

Anything with units I'm fairly sure was taken as-is from the dashboard template of the Material design plugin thing :D Good point though, not sure why it is as it is.

**Pull request merged**
